README.md

```diff
@@ -36,4 +36,21 @@ When `semantic-release` runs, this plugin will update the following files accord
 - `package.json`
 - `package-lock.json`
 - `manifest.json`
+- `manifest-beta.json`
 - `versions.json`
+
+## Beta Release Support with BRAT
+
+When running `semantic-release` on a pre-release branch (e.g. `beta`), the plugin updates only `manifest-beta.json` and `versions.json` with the beta-version number, preserving `manifest.json`. This enables [BRAT](https://github.com/TfTHacker/obsidian42-brat) compatibility.
```

Comment:
I don't see `beta` hard-coded anywhere, does that mean _any_ branch name would work? If so, please update to communicate that.